Library/Homebrew/utils/bottle_api.rb

```diff
@@ -0,0 +1,103 @@
+# typed: true
+# frozen_string_literal: true
+
+require "github_packages"
+
+module Utils
```

Comment:
I wonder if this should be in `Utils` or just its own class?

Reply:
No preference. I put it in `Utils` originally thinking it would be a helper module for using the `/api/bottle` API, but since it does contain a few methods that aren't really directly related to the API I would be fine moving it to its own module.